index.html

```diff
@@ -29,22 +30,33 @@
             <h2>Marwan</h2>
           </a>
         </div>
-        <div class="bars">
+        <div class="bars" id="toggle-icon" onclick="toggleMobileMenu(this)">
           <div class="bar1"></div>
           <div class="bar2"></div>
           <div class="bar3"></div>
+          <ul class="nav-menu">
+            <li>
+              <a href="https://github.com/MarwanDev">Portfolio</a><hr>
+            </li>
+            <li>
+              <a href="https://github.com/MarwanDev">About</a><hr>
+            </li>
+            <li>
+              <a href="https://github.com/MarwanDev">Contact</a><hr>
+            </li>
```

Comment:
- [ ] I can see you did a perfect implementation on this project. However, this part of your code should lead to the place that is only inside your website. Portfolio => Portfolio section, About => About section, Contact => Contact section